clients/admin-ui/src/pages/add-systems/manual.tsx

```diff
@@ -46,27 +45,9 @@ const NewManualSystem: NextPage = () => {
 
   return (
     <Layout title="Describe your system">
-      <Box mb={4}>
-        <Header connector={connector} />
-        <Box>
-          <Breadcrumb
-            fontWeight="medium"
-            fontSize="sm"
-            color="gray.600"
-            data-testid="breadcrumbs"
-          >
-            <BreadcrumbItem>
-              <NextLink href={systemOrDatamapRoute}>Data map</NextLink>
-            </BreadcrumbItem>
-            <BreadcrumbItem>
-              <NextLink href={ADD_SYSTEMS_ROUTE}>Add systems</NextLink>
-            </BreadcrumbItem>
-            <BreadcrumbItem color="complimentary.500">
-              <NextLink href="#">Describe your system</NextLink>
-            </BreadcrumbItem>
-          </Breadcrumb>
-        </Box>
-      </Box>
+      <BackButton backPath={systemOrDatamapRoute} />
```

Comment:
Is this the desired back button behavior? The add system pages are doing back to the datamap instead of the base add system page.


https://github.com/ethyca/fides/assets/17103888/5dc2506f-8aba-46e5-ab23-879deadadf57

Reply:
ah yeah that's a good point! I was copying the original breadcrumb behavior but I think that logic worked better for breadcrumbs than it does for back buttons. I'll fix!